packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -705,6 +705,61 @@ login:
     expires: never
     data_sensitivity:
       - interaction
+  login_backup_code_view:
```

Comment:
@chenba these new events are under `login` category (line 443 above). If we keep their names here prefixed with `login_`, they will be collected and visible in BQ as `login.login_backup_code_*`.
We might want to consider removing `login_` prefix from the names.